[request.py]
  [test_provide_response_auth_required[fetch\]]
    expected: FAIL

  [test_provide_response_auth_required[navigate\]]
    expected: FAIL

  [test_provide_response_phase[fetch-beforeRequestSent\]]
    expected: FAIL

  [test_provide_response_phase[fetch-responseStarted\]]
    expected: FAIL

  [test_provide_response_phase[navigate-beforeRequestSent\]]
    expected: FAIL

  [test_provide_response_phase[navigate-responseStarted\]]
    expected: FAIL
